tools/ci-scripts/check-aws-sdk-adhoc-tests

```diff
@@ -4,6 +4,17 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-set -eux
+C_YELLOW='\033[1;33m'
+C_RESET='\033[0m'
+
+set -eu
 cd smithy-rs
-./gradlew aws:sdk-adhoc-test:test
+
+# TODO(enableNewSmithyRuntime): Remove the middleware test run when cleaning up middleware
+echo -e "## ${C_YELLOW}Running SDK adhoc tests against the middleware implementation...${C_RESET}"
+./gradlew aws:sdk-adhoc-test:clean
+./gradlew aws:sdk-adhoc-test:check -Psmithy.runtime.mode=middleware
+
+echo -e "## ${C_YELLOW}Running SDK adhoc tests against the orchestrator implementation...${C_RESET}"
+./gradlew aws:sdk-adhoc-test:clean
+./gradlew aws:sdk-adhoc-test:check -Psmithy.runtime.mode=orchestrator
```

Comment:
does this work cleanly in GitHub actions re: the coloring?

Reply:
It has worked for the other scripts that do it.